src/song.rs

```diff
@@ -118,8 +118,9 @@ impl Encodable for Song {
         e.emit_struct("Song", 7, |e| {
```

Comment:
Please set number of fields to 8. Otherwise I'm ready to accept the PR. Thank you!